data-files/samples/weapons.Experiment.Any

```diff
@@ -0,0 +1,151 @@
+{ 
+    description = "manytargets"; 
+    frameRate = 120; 
+    showHUD = true;
+    renderWeaponStatus = true;
+    cooldownMode = "ring";
+    cooldownColor = Color4(0.7,1.0,0.9,0.75);
+    sessions = ( 
+        { 
+            description = "1 shot"; 
+            id = "1shot"; 
+            settingsVersion = 1; 
```

Comment:
Again settings version really doesn't _need_ to be specified here, but if it is I usually do `id, description, settings version` at the top of the session...